app/javascript/bulk.js

```diff
@@ -29,15 +29,6 @@ function process_patch(druids, action_url, req_params, success_string) {
 	process_request(druids, action_url, 'PATCH', req_params, success_string, show_buttons, show_buttons);
 }
 
-function set_content_type(druids){
-	var params={
-		'new_content_type': document.getElementById('new_content_type').value,
-		'new_resource_type': document.getElementById('new_resource_type').value,
-		'old_resource_type': document.getElementById('old_resource_type').value
-	}
-	process_patch(druids, set_content_type_url, params, "Updated");
```

Comment:
Can you remove the `process_patch` function too?

Reply:
Ah, thanks for catching that, have removed `process_patch`.